modules/nextflow/src/main/groovy/nextflow/scm/ProviderConfig.groovy

```diff
@@ -274,7 +274,7 @@ class ProviderConfig {
     static Map getFromFile(Path file) {
         try {
             final content = file.text
-            log.trace "Parsing SCM config path: ${file.toUriString()}\n${content}\n"
+            log.trace "Parsing SCM config path: ${file.toUriString()}\n${StringUtils.stripSecrets(content)}\n"
```

Comment:
let's wrap into a separate method like this 

https://github.com/nextflow-io/nextflow/blob/c3117ada3e4e2b661baf6c3456cd9f2c092be85e/modules/nextflow/src/main/groovy/nextflow/scm/ProviderConfig.groovy#L303-L310

I've seen StackOverflow exception thrown by regex recently 🤦. better to be sure there's any side effect for this logging